src/components/AppPostCard.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div class="app-post-card full-width row rounded-borders full-height q-pa-md q-mb-md shadow-up-8">
+    <div class="app-post-card__image col-12 col-md-6">
+      <q-badge class="app-post-card__category absolute text-weight-bold text-white bg-red q-mt-md q-py-sm q-px-md">
+        {{ props.category }}
+      </q-badge>
+
+      <q-img
+        :src="props.image"
+        :ratio="16/9"
+      />
+    </div>
+
+    <div class="col-12 col-md-6 column justify-between">
+        <!-- Title -->
+        <div class="app-post-card__content full-width column col-9 q-pa-sm">
+          <div class="full-width col-3 bg-light-gray rounded-borders content-center">
+            <h4 class="ellipsis"> {{ props.title }} </h4>
```

Comment:
remover esse espaço entre o titulo, se for pra usar espaço assim, usar margin/padding